android/app/build.gradle

```diff
@@ -55,8 +54,8 @@ android {
         applicationId "com.example.gameonconnect"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion localProperties.getProperty('flutter.minSdkVersion').toInteger()
-        targetSdkVersion localProperties.getProperty('flutter.targetSdkVersion').toInteger()
```

Comment:
Hi Franco, this should not be changed back, it will create issues with the code. Think you might have mistakenly changed this when merging?  The code should stay as: 

minSdkVersion localProperties.getProperty('flutter.minSdkVersion').toInteger()
targetSdkVersion localProperties.getProperty('flutter.targetSdkVersion').toInteger()